src/main/java/pl/put/poznan/transformer/logic/TextTransformer.java

```diff
@@ -18,6 +18,51 @@ public String transform(String text){
         // of course normally it would to something based on transforms
         return text.toUpperCase();
     }
+    
+    /**
+     * This function is used for transformation all letters to upper case.
+     * @param text - Input string that will be changed
+     * @return string where all letters in upper case
+     */
+    static String upper(String text)
+    {
+	 return text.toUpperCase();
+    }
+
+    /**
+     * This function is used for transformation all letters to lower case.
+     * @param text - Input string that will be changed
+     * @return string where all letters in lower case
+     */
+    static String lower(String text)
+    {	
+         return text.toLowerCase();
+    }
+	
+    /**
+     * This function is used for transformation onle the first letter in each sentence to upper case.
+     * @param text - Input string that will be changed
+     * @return string where onle the first letter in each sentence in upper case
+     */
+    static String capital(String text)
+    {
+          char lit[] = text.toCharArray();
```

Comment:
Nie zmieni to działania kodu, ale myślę, że patrząc na czytelność kodu warto wyrównać tę linię z innymi na jej poziomie (1 tab mniej)